.github/ISSUE_TEMPLATE/FEATURE_REQUEST.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Feature Request
+about: Suggest a feature or a new component for LPL's component library
+labels: enhancement
+---
+
+## Feature Description
+
```

Comment:
Maybe ask the author to describe the problem that the feature solves as part of this description.